pages/en/blog/vulnerability/june-2023-security-releases.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2023-06-13T17:00:00.000Z
```

Comment:
@RafaelGSS do we want this to be closer to now? From my check it would only show up ~4 hours from now.